components/TheImageUploadForm.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <transition name="the-upload-image-form--slide-fade">
```

Comment:
`the-upload-image-form-` for transition name would suffice

Reply:
Besides, I see there is no v-if props inside its child component, should put the transition in upload page instead??